src/api/v1beta1/dynakube_status.go

```diff
@@ -54,9 +56,29 @@ type DynaKubeStatus struct {
 	// Conditions includes status about the current state of the instance
 	Conditions []metav1.Condition `json:"conditions,omitempty"`
 
-	ActiveGate ActiveGateStatus `json:"activeGate,omitempty"`
-	OneAgent   OneAgentStatus   `json:"oneAgent,omitempty"`
-	Synthetic  SyntheticStatus  `json:"synthetic,omitempty"`
+	ActiveGate   ActiveGateStatus   `json:"activeGate,omitempty"`
+	OneAgent     OneAgentStatus     `json:"oneAgent,omitempty"`
+	Synthetic    SyntheticStatus    `json:"synthetic,omitempty"`
+	DynatraceApi DynatraceApiStatus `json:"dynatraceApi,omitempty"`
+}
+
+const MaxRequestInterval = 15 * time.Minute
+
+type DynatraceApiStatus struct {
+	LastTokenProbe                     metav1.Time `json:"lastTokenProbe,omitempty"`
```

Comment:
suggestion: out of consistency maybe rename this to `LateTokenProbeTimestamp`?

Reply:
IMO `Timestamp` is redundant
but the rest of the names could be more consistent. (replace `...Update` with `Request` and then it could be called `LastTokenScopeRequest`)